android/app/src/main/kotlin/com/cBreez/client/BreezForegroundService.kt

```diff
@@ -39,8 +39,8 @@ class BreezForegroundService : ForegroundService() {
                 "ERROR" -> Logger.tag(TAG).error { l.line }
                 "WARN" -> Logger.tag(TAG).warn { l.line }
                 "INFO" -> Logger.tag(TAG).info { l.line }
-                "DEBUG" -> Logger.tag(TAG).debug { l.line }
-                "TRACE" -> Logger.tag(TAG).trace { l.line }
+                // "DEBUG" -> Logger.tag(TAG).debug { l.line }
+                // "TRACE" -> Logger.tag(TAG).trace { l.line }
```

Comment:
Do these need reseting?

Reply:
No, it's continuation of
- #817 
but I'll address it as part of another PR.